src/query/pipeline/sources/src/input_formats/input_format_text.rs

```diff
@@ -599,3 +653,8 @@ fn decompress(decoder: &mut DecompressDecoder, compressed: &[u8]) -> Result<Vec<
     }
     Ok(decompress_bufs.concat())
 }
+
+fn is_splittable(info: &StageFileInfo) -> bool {
+    (info.path.ends_with("csv") || info.path.ends_with("tsv"))
```

Comment:
1. currently not based on suffix
2. overlap with T::is_splittable()

Reply:
yes, please let me revert my split way first, that's not correct

Reply:
I **highly** suggest not to do split first, distributed files first.

Reply:
We do not recommend users to copy large files, this is not the case for Databend now.

Reply:
yes